### PR TITLE
I29 upload csv 2

### PR DESCRIPTION
**Pull request by jpengst**
Finished csv upload with radio buttons to either file upload or determine file path on server.

### CODE REVIEW

File: app/views/bulkrax/importers/_csv_fields.html.erb [hunk 1]
```diff
@@ -8,5 +7,17 @@
...
+
+  <h4>Add CSV File to Import:</h4>
+  <%# accept a single file upload; data files and bags will need to be added another way %>
+  <%# @todo - for updates show the currently uploaded file and make it clear it will get replaced %>
+  <%= fi.input :file_style, collection: ['Upload a File', 'Specify a Path on the Server'], as: :radio_buttons, label: false %>
+  <div id='file_upload'>
+    <%= fi.input 'file', as: :file, input_html: {accept: 'text/csv'} %><br />
+  </div>
+  <div id='file_path'>
+    <%= fi.input :import_file_path, as: :string, input_html: { value: importer.parser_fields['import_file_path'] } %>
```
[the remark this answers is not in the document]
Comment (orangewolf) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: yes, i agree Julie. making it default to file_path on edit makes sense.

File: app/views/bulkrax/importers/_csv_fields.html.erb [hunk 2]
```diff
@@ -8,5 +7,17 @@
...
         input_html: { class: 'form-control' } %>
   <%= fi.input :override_rights_statement, as: :boolean, hint: 'If checked, always use the selected rights statment. If unchecked, use rights or rights_statement from the record and only use the provided value if dc:rights is blank.', input_html: { checked: (importer.parser_fields['override_rights_statement'] == "1") } %>
-  
+
+  <h4>Add CSV File to Import:</h4>
+  <%# accept a single file upload; data files and bags will need to be added another way %>
+  <%# @todo - for updates show the currently uploaded file and make it clear it will get replaced %>
+  <%= fi.input :file_style, collection: ['Upload a File', 'Specify a Path on the Server'], as: :radio_buttons, label: false %>
+  <div id='file_upload'>
+    <%= fi.input 'file', as: :file, input_html: {accept: 'text/csv'} %><br />
```
[the remark this answers is not in the document]
Comment (orangewolf) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: in @jpengst and my testing they were hidden via css. it sounds like maybe that got broken? @jpengst can you confirm when you load the form if the divs are showing or hidden?

File: app/views/bulkrax/importers/_csv_fields.html.erb [hunk 3]
```diff
@@ -8,5 +7,17 @@
         item_helper: rights_statements.method(:include_current_value),
         input_html: { class: 'form-control' } %>
   <%= fi.input :override_rights_statement, as: :boolean, hint: 'If checked, always use the selected rights statment. If unchecked, use rights or rights_statement from the record and only use the provided value if dc:rights is blank.', input_html: { checked: (importer.parser_fields['override_rights_statement'] == "1") } %>
-  
+
+  <h4>Add CSV File to Import:</h4>
+  <%# accept a single file upload; data files and bags will need to be added another way %>
+  <%# @todo - for updates show the currently uploaded file and make it clear it will get replaced %>
```
[the remark this answers is not in the document]
Comment (orangewolf) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: I think Julie means above, but yes, i agree we can cross this off the list

File: app/views/bulkrax/importers/_csv_fields.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jpengst) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: Hmm. Yeah, it seems to be working fine on my machine. They are both hidden on initial load and then render based on which radio button I click.

File: app/views/bulkrax/importers/_csv_fields.html.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (geekscruff) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: I do, sorry!

File: app/views/bulkrax/importers/_csv_fields.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (geekscruff) [reply app/views/bulkrax/importers/_csv_fields.html.erb]: That's likely build / cacheing on my end then. Sorry!

**Pull request merged**